charts/pulsar/values.yaml

```diff
@@ -228,29 +228,48 @@ tls:
     keySize: 4096
     keyAlgorithm: RSA
     keyEncoding: PKCS8
+    caSecretName:
   # settings for generating certs for proxy
   proxy:
     enabled: false
     cert_name: tls-proxy
     createCert: true  # set to false if you want to use an existing certificate
+    # specify name of secret contain certificate if using pre-generated certificate
+    certSecretName:
+    untrustedCa: true
```

Comment:
Please add documentation comments for the added `untrustedCa` configuration keys. I wonder if this is the best possible name for the configuration key. Please consider a better name, for example `mountKeyAndCertificate` etc. if that's what this is about.

Reply:
updated the description in https://github.com/apache/pulsar-helm-chart/pull/559/commits/20ed697afc5659168610a46241e5252fe4ecb3f6